src/app/components/ContentContainer/ContentContainer.tsx

```diff
@@ -4,16 +4,18 @@ interface ContentContainerProps {
   children: React.ReactNode;
   align?: boolean;
   classes?: string;
+  sectionClasses?: string;
 }
 export function ContentContainer({
   children,
   align = false,
   classes = '',
+  sectionClasses = '',
```

Comment:
I don't think we need to change anything here in this PR, but I was doing something the other day that made me think `ContentContainer` is a little too restrictive with what we can pass it. I may end up taking another look at adjusting it a bit to be easier to work with since it looks like you're hitting similar issues.